integration/log_test.go

```diff
@@ -0,0 +1,30 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/naggie/dstask"
+	"gotest.tools/assert"
+)
+
+func TestLog(t *testing.T) {
+	t.Skip("skipping because show-resolved cannot be run non-interactively yet")
```

Comment:
Skipping this for now. We could remove skip once we can run show-resolved w/o a TTY.

Reply:
What's the reason we can't? Just not implemented, I assume.

Reply:
Correct, merely that it isn't implemented yet